src/main/java/delete_task_use_case/DeleteTaskPresenter.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Delete Task Presenter Class.
+ * Author: Ricky Fung
+ * Modified By: Ricky Fung
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 28, 2022
+ */
+package delete_task_use_case;
+
+public class DeleteTaskPresenter implements DeleteTaskOutputBoundary {
+    @Override
+    public DeleteTaskOutputData prepareSuccessView(DeleteTaskOutputData taskOutputData) {
+        taskOutputData.setSuccess(true);
+        return taskOutputData;
+    }
```

Comment:
Yes we know delete theoretically won't ever have a fail view however we will add one to deal with issues that may come up when we implement GUI integration. along our use cases. Essentially please implement a fail view that provides an error message or anything that can let us realize that use case failed during dev/testing process.

Reply:
Added preparefailView method, to help us deal with issues that may come up during dev/testinng process.